frontend/src/App.js

```diff
@@ -11,29 +11,35 @@ import NewProject from './pages/newproject.js';
 import Style from './pages/style.js';
 import UserProfile from './components/UserProfile/Profile.component';
 import Profile from './components/Profile/Profile.component';
+import Footer from './pages/footer';
 import './App.css';
 import './about.css';
 
 function App() {
-	return (
-		<Router>
-			<div className="App">
-				<NavBar />
-				<Switch>
-					<Route path="/" exact component={HomePage} />
-					<Route path="/about" exact component={AboutPage} />
-					<Route path="/search" exact component={SearchPage} />
-					<Route path="/signup" exact component={SignupPage} />
-					<Route path="/signup/volunteer" exact component={SignupVolunteerPage} />
-					<Route path="/storybook" exact component={StoryBook} />
-					<Route path="/newproject" exact component={NewProject} />
- 					<Route path='/userprofile' exact component={UserProfile} />
-          <Route path="/profile" exact component={Profile} />
-					<Route path="/style" exact component={Style} />
-				</Switch>
-			</div>
-		</Router>
-	);
```

Comment:
It seems like you've some automatic code formatting in place. The diff is really hard to read, and increases chances for errors.

Would you mind changing this file so that you won't touch the formatting? Add only your changes and leave the formatting on later stages. You can use `git diff` to see what have changed before making a commit.

Appreciate your input nevertheless, keep on giving. We'll get there!

Reply:
Gosh, it's guess it must be Prettier format I am using that is causing this. I need to be more aware. Will fix it right away